libraries/Microsoft.Bot.Schema/Teams/Generated/TeamInfo.cs

```diff
@@ -55,5 +55,10 @@ public TeamInfo(string id = default(string), string name = default(string))
         [JsonProperty(PropertyName = "name")]
         public string Name { get; set; }
 
+        /// <summary>
+        /// Gets or sets aadGroupId of team.
```

Comment:
Can we reword this as Gets or sets the Azure AD group ID?

Reply:
changed the comment